app/assets/stylesheets/application.sass.scss

```
@@ -89,42 +93,45 @@ $color_app-pale-blue: #ccdff1;
   }
 }
 
-.app-header__navigation-text {
-  @include nhsuk-font(16);
-  border-bottom: nhsuk-spacing(1) solid transparent;
-  border-top: nhsuk-spacing(1) solid transparent;
-  color: $color_nhsuk-blue;
-  display: block;
-  padding: 12px nhsuk-spacing(3);
-  text-decoration: none;
-
-  @include mq($from: large-desktop) {
-    color: $color_nhsuk-white;
-    line-height: normal;
+.app-header {
+  &__container {
+    border-bottom: 1px solid rgba(255, 255, 255, 0.2);
   }
 
-  .nhsuk-icon__chevron-right {
-    fill: $color_nhsuk-grey-3;
-    position: absolute;
-    right: nhsuk-spacing(1);
-    top: 11px;
+  &__service-name {
+    display: inline-block;
+    line-height: 1.25;
+    vertical-align: top;
+
+    @include mq($until: tablet) {
+      padding-left: nhsuk-spacing(3);
+    }
+
+    @include mq($from: tablet, $until: large-desktop) {
+      max-width: 300px;
+    }
```

Am reviewing this blind; we want no left padding while it is under the logo on `mobile`, then reinstate it from `tablet`, right? So maybe it should be: 

```suggestion
  &__service-name {
    display: inline-block;
    line-height: 1.25;
    padding-left: 0;
    vertical-align: top;

    @include mq($from: tablet) {
      padding-left: nhsuk-spacing(3);
    }

    @include mq($from: tablet, $until: large-desktop) {
      max-width: 300px;
    }
```

Does that work?